openc3/python/test/packets/parsers/test_packet_item_parser.py

```diff
@@ -129,22 +129,27 @@ def test_only_allows_derived_items_with_offset_0_and_size_0(self):
     def test_accepts_types_int_uint_float_string_block(self):
         tf = tempfile.NamedTemporaryFile(mode="w")
         tf.write('TELEMETRY tgt1 pkt1 LITTLE_ENDIAN "Description"\n')
-        tf.write("  ID_ITEM ITEM1 0 32 INT 0\n")
+        tf.write("  ID_ITEM ITEM1 0 32 INT 0x42\n")
```

Comment:
This case currently fails in Python (but not Ruby) because Ruby calls `Integer(value)` which works ... the Python conversion code did not handle it.